### ISSUE TITLE
thumbnails can get stuck on top of interface

### ISSUE THREAD

**Issue by phivk** [unfoldcollective/Apprentice-of-Okolicne#106]
EDIT: possibly caused by palette touched with multiple fingers? 

I witnessed this twice in a row, will investigate and try to document how to reproduce more precisely

e.g. background element stays in thumbnail shape and is 'locked' in position on top of canvas / palette. navigating back to welcome screen doesn't remove the thumbnail.

**Comment by phivk** [comment]
![img_4392](https://user-images.githubusercontent.com/902958/30324482-7ecc4776-97c1-11e7-95aa-556d60307460.JPG)

**Comment by phivk** [comment]
apparently can also happen with objects:
![img_4394](https://user-images.githubusercontent.com/902958/30324509-98fc7512-97c1-11e7-808b-e02b63227e19.JPG)

**Comment by phivk** [comment]
console view just after pattern thumbnail got stuck:
![screen shot 2017-09-14 at 17 14 36](https://user-images.githubusercontent.com/902958/30439169-558fbede-9973-11e7-92f0-d4ba5150bf71.png)

**Comment by phivk** [comment]
@bertez after calling with testers and viewing via team viewer I strongly believe that this is the main issue and other issues #111 and #108 are side effects of this issue. 

What I mean is that the way to cause all these issues is to 'multi touch' thumbnails in the pallete (and perhaps drag them a little). From that moment one or more thumbnails get stuck and also the behaviour of other elements is effected:
- the drag delay affects other thumbnails
- elements on the canvas can not be moved, rotated or scaled (they can be flipped and brought to front)

In terms of bugs, let's focus our attention on this issue

**Comment by phivk** [comment]
screen recordings with console view: https://photos.app.goo.gl/VGa1Gpor2HzDPApz1

**Comment by phivk** [comment]
Today I hunted this issue down to the combination of the Windows PC in use + the Iiyama touchscreen.

The following setups work:
- mac osx laptop + Iiyama touch screen ✔️ 
- downstairs Windows PC + BenQ touch screen ✔️ 

the issue being related to the combination of Iiyama touch screen and Windows PC configuration is corroborated by Iiyama technical staff, who advice to try changing driver setup (see #114 )

In my eyes there are 3 potential solutions left (for SNG to explore):
1. change / update driver of used PC according to instructions from Iiyama ( #114 ) 
2. exchange currently used PC for another computer.
3. exchange the Iiyama screen and the BenQ in use downstairs (tested in combination with the PC downstairs)

Closing this issue as potential solutions are out of our hands.
